src/views/Pool.vue

```diff
@@ -235,23 +281,28 @@ export default {
     async borrowHandler(data) {
       console.log("BORROW HANDLER", data);
 
-      const isTokenApprove = await this.isTokenApprowed(
-        this.pool.token.contract,
-        this.pool.masterContractInstance.address
-      );
-
       const isApprowed = await this.isApprowed();
+      const useAVAXStatus = this.getAVAXStatus();
+
+      if (!useAVAXStatus) {
```

Comment:
you can reverse the logic in this condition, to remove unnecessary block spacing and make code more readable

Reply:
Done